apps/site/src/lib/components/Button/Button.module.scss

```diff
@@ -13,10 +13,13 @@
 	display: inline-block;
 	background-color: white;
 
-	&:hover > .button {
+	& > .button {
 		transition: all 0.15s ease-out;
-		transform: translate(-8px, -8px);
-		filter: brightness(1.05);
+
+		&:hover {
+			transform: translate(-8px, -8px);
+			filter: brightness(1.05);
+		}
```

Comment:
Hover effect is a bit glitchy when mouse is moved with the Button moving back down to its original position and animating back up